gst/nnstreamer/include/nnstreamer_plugin_api.h

```diff
@@ -53,10 +53,12 @@ gst_tensors_config_from_structure (GstTensorsConfig *config,
  * @brief Parse caps and set tensors config (for other/tensors)
  * @param[out] config tensors config structure to be filled
  * @param[in] caps incoming capability
+ * @param[in] validate TRUE to validate configuration
  * @return TRUE/FALSE (if successfully configured, return TRUE)
  */
 extern gboolean
-gst_tensors_config_from_caps (GstTensorsConfig *config, const GstCaps *caps);
+gst_tensors_config_from_caps (GstTensorsConfig *config, const GstCaps *caps,
```

Comment:
What is the reason for validation check with param?

Reply:
For the flexibility to parse caps, tensor information is not fully configured.

Reply:
Tensors config is obtained from the fixed cap. Is it necessary to obtain tensor information that is not completely configured?

Reply:
query-caps event may parse unfixed caps. (e.g. transform_caps() callback from base-transform in tensor-filter)